public/app/plugins/panel/graph/graph.ts

```diff
@@ -185,13 +185,23 @@ module.directive('grafanaGraph', function($rootScope, timeSrv) {
 
       // Series could have different timeSteps,
       // let's find the smallest one so that bars are correctly rendered.
+      // In addition, only take series which are rendered as bars for this.
       function getMinTimeStepOfSeries(data) {
         var min = Number.MAX_VALUE;
 
         for (let i = 0; i < data.length; i++) {
           if (!data[i].stats.timeStep) {
             continue;
           }
+          if (panel.bars) {
+            if (typeof data[i].bars !== 'undefined' && typeof data[i].bars.show !== 'undefined' && !data[i].bars.show) {
```

Comment:
If `data[i].bars.show` has not been defined by the user (then false "by default"), `data[i].bars.show` will be undefined and it will break the tests with :
`undefined is not an object (evaluating 'data[i].bars.show')`
(see previous failed tests)